src/cpp/function.h

```diff
@@ -26,8 +26,9 @@ class Function : public GCObject<FunctionData> {
     sol::object convert(lua_State* state, const Converter& clbk) const;
     void construct(const sol::function& luaObject, SolTableToShared& visited);
 
-    explicit Function(const sol::function& luaObject, SolTableToShared& visited);
-    explicit Function(const sol::function& luaObject);
+    Function() = default;
+    void initialize(const sol::function& luaObject, SolTableToShared& visited);
```

Comment:
You can just rename `contruct` to `initialize`.
`construct` and `initialize` both present in class confuse me.

Reply:
Yep, no we can